content/en/blog/2023/sunsetting-opencensus.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Sunsetting OpenCensus
```

Comment:
Is there any announcement from the OpenCensus official site? It'll be great to have the cross reference.

Reply:
We're planning to update the [banner](https://github.com/census-instrumentation/opencensus-website/blob/5970b0c5598460f0a22ec994826b7a0962352490/content/_index.md?plain=1#L10) on the OpenCensus website to announce the date and point with a link to this blog post.